packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -61,9 +61,10 @@
     "@itwin/imodels-access-common": "^4.1.2",
     "@itwin/imodels-access-frontend": "^4.1.2",
     "@itwin/imodels-client-management": "^4.2.1",
-    "@itwin/insights-client": "^0.4.0",
+    "@itwin/insights-client": "^0.6.0",
```

Comment:
I think this means that this whole PR can no longer be a `patch`. Pre-release (>1.0.0) minor version changes can have breaking changes, therefore this package upgrade might be a breaking change for some users.